src/models/GllobalValidations/DescriptionValidation.tsx

```diff
@@ -239,4 +239,14 @@ export const descriptionValidation = ({
             message: emptyInput(),
         },    
     ],
+    DescriptionNotOnlyWhiteSpaces: [
+        {
+            pattern: /^\s*\S.*$/,
+            message: inputOnlyWhiteSpaces(),
+        },
+        { 
+            max: 1000, 
+            message: maxLength(1000),
+        },    
+    ],
```

Comment:
Maybe it is better to change the previous method, because it looks almost the same?

Reply:
can I delete description method?

Reply:
Done

Reply:
I was told about DescriptionAndQuestions Method) Description was using at some another place

Reply:
misunderstanding) Description was used only there where I changed it to  DescriptionNotOnlyWhiteSpaces

Reply:
DescriptionAndQuestions also says that field is mandatory, but we don't need this in description city/hovel/...